draft-ietf-webpush-protocol.xml

```diff
@@ -305,7 +305,22 @@
         </t>
       </section>
     </section>
-
+    <section title="Connecting to the Push Service">
+      <t>
+        The push service shares the same default port number (443/TCP) with HTTPS, but MAY
+        also advertise the IANA allocated TCP port 1001 using HTTP alternative services

```

Comment:
OK. I was responding to this requirement in RFC6335:

o  Port Number: If assignment of a port number is desired, either the
      port number the requester suggests for assignment or indication of
      port range (user or system) MUST be provided.